src/main/java/com/example/cosmocats/repository/entity/CustomerEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.cosmocats.repository.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "customers")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CustomerEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
```

Comment:
I explained a bit why it's not the best solution here https://www.youtube.com/watch?v=mfvvQIWv2Zk&t=977s